src/database/types.ts

```diff
@@ -81,6 +81,7 @@ export type CreateCollectionStoryInput = {
   authors: CollectionStoryAuthor[];
   publisher: string;
   sortOrder?: number;
+  fromPartner: boolean;
```

Comment:
as this defaults to `false` in the prisma schema, and most collections will _not_ be sponsored, what do you think of making this value optional? thinking it would be nice to only need this value when working with a sponsored collection.

Reply:
It crossed my mind - but then I looked up all the other inputs and it seems that even with a default value if it's a required field it will be a required field in the input type. A recent example of that is the new `language` field on the `Collection` type - defaults to `en` in Prisma schema yet is required in both `CreateCollectionInput` and `UpdateCollectionInput`. So I made the `fromPartner` field required, too, for consistency.

Reply:
@nina-py I actually agree with @jpetto here, if we make this optional for now we can deploy this change and not need to time it with the admin tool. 

Now for consistency, nothing says we can't follow up and make it required later too. We should try and decouple systems/releases from each other as much as we can.

Reply:
I agree if it's the best way to avoid being blocked/having downtime at deployment time. I've made the change and added a couple of integration tests, too.